Examples/create-app.js

```diff
@@ -152,14 +171,17 @@ function optimizeToTestInDebugMode() {
         }
     } catch(e) {}
     
+    let rnXcodeShPath = `node_modules/react-native/${rnXcodeShLocationFolder}/react-native-xcode.sh`;
+    // Replace "if [[ "$PLATFORM_NAME" == *simulator ]]; then" with "if false; then" to force bundling
+    execSync(`sed -ie 's/if \\[\\[ "\$PLATFORM_NAME" == \\*simulator \\]\\]; then/if false; then/' ${rnXcodeShPath}`);
+    
     execSync(`perl -i -p0e 's/#ifdef DEBUG.*?#endif/jsCodeLocation = [CodePush bundleURL];/s' ios/${appName}/AppDelegate.m`);
-    execSync(`sed -ie '17,20d' node_modules/react-native/${rnXcodeShLocationFolder}/react-native-xcode.sh`);
-    execSync(`sed -ie 's/targetName.toLowerCase().contains("release")$/true/' node_modules/react-native/react.gradle`);
+    execSync(`sed -ie 's/targetName.toLowerCase().contains("release")/true/' node_modules/react-native/react.gradle`);
 }
 
 function grantAccess(folderPath) {
     execSync('chown -R `whoami` ' + folderPath);
-    execSync('chmod -R 755 ' + folderPath);
+    // execSync('chmod -R 755 ' + folderPath);
```

Comment:
Could you please provide the exact error/problem caused by this line? It works on mac previously so just wondering what happens now? 
Also if it turns out that we really do not need this it make sense to remove this like and not to comment it.

Reply:
I guess that it was my system specific issue and I can't reproduce it now, so I think it's better to just return this code.